modules/opengl/src/buffer/framebufferobject.cpp

```diff
@@ -94,6 +114,8 @@ FrameBufferObject& FrameBufferObject::operator=(FrameBufferObject&& rhs) noexcep
 }
 
 FrameBufferObject::~FrameBufferObject() {
+    checkContext("FBO deleted on a different context"sv, creationContext_, IVW_SOURCE_LOCATION);
```

Comment:
We could provide more info here. Something like:
FBO deleted on a different context. Did you use the FBO in a different thread from where it was created?

Reply:
It does print the name and if of the created context and the current one.